diff -rupN libopenmpt-0.8.2+release.autotools.old/configure.ac libopenmpt-0.8.2+release.autotools/configure.ac
--- libopenmpt-0.8.2+release.autotools.old/configure.ac	2025-07-19 15:10:54.000000000 +0200
+++ libopenmpt-0.8.2+release.autotools/configure.ac	2025-07-25 14:03:24.943377747 +0200
@@ -1,4 +1,4 @@
-AC_INIT([libopenmpt], [0.8.2+release.autotools], [https://bugs.openmpt.org/], [libopenmpt], [https://lib.openmpt.org/])
+AC_INIT([libopenmpt], [0.8.2], [https://bugs.openmpt.org/], [libopenmpt], [https://lib.openmpt.org/])
 AC_PREREQ([2.69])
 # we do want 2.70, but distributions are not ready yet
 #AC_PREREQ([2.70])
